src/async_fuse/fuse/session.rs

```diff
@@ -104,8 +104,11 @@ impl Drop for FuseFd {
 
 impl Drop for Session {
     fn drop(&mut self) {
-        smol::block_on(async {
-            futures::future::join_all(self.tasks.drain(..).map(smol::Task::cancel)).await;
+        futures::executor::block_on(async {
```

Comment:
Maybe tokio::runtime::new().block_on()  is better?

Reply:
cannot create runtime within a runtime